### PR TITLE
added styles from corebundle

### PR DESCRIPTION
**Pull request by silasjoisten**
Adding a few styles from corebundle

### CODE REVIEW

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -396,6 +396,8 @@ public function getConfigTreeBuilder()
                                 'bundles/sonataadmin/css/styles.css',
                                 'bundles/sonataadmin/css/layout.css',
                                 'bundles/sonataadmin/css/tree.css',
+
+                                'bundles/sonatacore/css/flashmessage.css',
```
Comment (core23) [inline src/DependencyInjection/Configuration.php]: This file will always exist? No change in the `composer.json`?

Comment (silasjoisten) [reply src/DependencyInjection/Configuration.php]: What do you mean? after the merge of the flashmessage stuff it will exists.

Comment (core23) [reply src/DependencyInjection/Configuration.php]: In every CoreBundle version?

Comment (silasjoisten) [reply src/DependencyInjection/Configuration.php]: Ah okay which version shall i use?

Comment (OskarStark) [reply src/DependencyInjection/Configuration.php]: We need to merge core bundle stuff, release it and then require this version as min version

**Comment by silasjoisten** [comment]
Okay i am done

**Comment by OskarStark** [comment]
@core23 please review again and merge, thank you!

**Comment by OskarStark** [comment]
@silasjoisten I can’t remember, can you please tell why this should be done?

**Comment by OskarStark** [comment]
Currently the batch flash messages are looking bad, this fixes the problem

**Pull request merged**